inc/mailcollector.class.php

```diff
@@ -717,9 +720,7 @@ function collect($mailgateID, $display = 0) {
                $rejinput                      = [];
                $rejinput['mailcollectors_id'] = $mailgateID;
 
-               $req_field = $this->getRequesterField();
-               $h_requester = $message->getHeader($req_field)->getAddressList();
-               $requester = $h_requester->current()->getEmail();
+               $requester = $this->getRequesterEmail($message);
```

Comment:
`reply-to` was always defined when using the imap extension (see https://github.com/glpi-project/glpi/pull/4554#discussion_r215509231), but now this header is not available if not set in the message.